src/main/webapp/app/exercises/quiz/shared/questions/drag-and-drop-question/drag-and-drop-question.component.ts

```diff
@@ -217,9 +217,11 @@ export class DragAndDropQuestionComponent implements OnChanges {
      */
     getUnassignedDragItems() {
         return this.question.dragItems?.filter((dragItem) => {
-            return !this.mappings.some((mapping) => {
-                return this.dragAndDropQuestionUtil.isSameDragItem(mapping.dragItem!, dragItem);
-            }, this);
+            return (
+                !this.mappings?.some((mapping) => {
+                    return this.dragAndDropQuestionUtil.isSameDragItem(mapping.dragItem!, dragItem);
+                }, this) ?? true
```

Comment:
I find
```suggestion
            return (
                !this.mappings || !this.mappings.some((mapping) => {
                    return this.dragAndDropQuestionUtil.isSameDragItem(mapping.dragItem!, dragItem);
                }, this);
```
a bit more readable, but that is just personal preference.